README.md

```diff
@@ -58,6 +58,7 @@ All the tasks are built using [gulp](http://gulpjs.com/), and almost all of them
 	   [--scssLintPath=<path>]      Custom scss-lint configuration
 	   [--esLintPath=<path>]        Custom esLint configuration
 	   [--editorconfigPath=<path>]  Custom .editorconfig
+	   [--babelRuntime=false]       Exclude Babel polyfills
```

Comment:
This should go in the build task section of the readme. Currently it's not a common option, so I´m not sure it should go here, but if you think we need to highlight it more, then sure, but you also need to update the cli.js file